### ISSUE TITLE
“reblogged” should be “boosted”

### ISSUE THREAD

**Issue by Scenario** [mastodon/mastodon#22289]
### Steps to reproduce the problem

Look at your timeline where people have boosted posts.

### Expected behaviour

soandso reblogged

### Actual behaviour

sosandso boosted

### Detailed description

Blogged and Reblogged are really stupid verbs. Mastodon is not a blogging site. On a blog site, commenters can’t reblog you on that some site. They can only comment in situ. What’s really happening here is that when people think you said something worthy, they can boost you. So that should be the verb.

### Specifications

Latest iOS app

**Comment by joenepraat** [comment]
Agreed, but wrong repository. See https://github.com/mastodon/mastodon-ios
